ctapipe/reco/event_classifier.py

```diff
@@ -3,6 +3,8 @@
 from astropy import units as u
 
 from sklearn.ensemble import RandomForestClassifier
+from sklearn.neural_network import MLPClassifier
+from sklearn.preprocessing import StandardScaler
 
```

Comment:
and -- concidering my other comment -- the `StandardScaler` import can also rather go to `RegressorClassifierBase`, no?